src/components/FormStart/tests.spec.js

```diff
@@ -112,12 +106,12 @@ it('Form start page does not show login buttons if an active submission is prese
         form={testForm}
         onFormStart={onFormStart}
         onDestroySession={onDestroySession}
-        hasActiveSubmission={true}
+        submission={{id: 'tralala', isAuthenticated: false}}
```

Comment:
This will cause prop type check warnings, please use the mock facilities to build a submission object that conforms to the prop types: https://github.com/open-formulieren/open-forms-sdk/blob/main/src/api-mocks/index.js#L3

`submission={buildSubmission({isAuthenticated: false})}` should be sufficient